src/styles/card.css

```diff
@@ -0,0 +1,86 @@
+.page {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__images {
+  margin: 0 3px 40px;
```

Comment:
check margin, how we can center the image without set 3px for horizontal margin?